packages/discord.js/src/structures/BaseGuildVoiceChannel.js

```diff
@@ -199,33 +217,46 @@ class BaseGuildVoiceChannel extends GuildChannel {
    *   .then(channel => console.log(`Set user limit to ${channel.userLimit} for ${channel.name}`))
    *   .catch(console.error);
    */
-  setUserLimit(userLimit, reason) {
+  async setUserLimit(userLimit, reason) {
     return this.edit({ userLimit, reason });
   }
 
   /**
    * Sets the camera video quality mode of the channel.
+   *
    * @param {VideoQualityMode} videoQualityMode The new camera video quality mode.
    * @param {string} [reason] Reason for changing the camera video quality mode.
    * @returns {Promise<BaseGuildVoiceChannel>}
    */
-  setVideoQualityMode(videoQualityMode, reason) {
+  async setVideoQualityMode(videoQualityMode, reason) {
     return this.edit({ videoQualityMode, reason });
   }
 
   // These are here only for documentation purposes - they are implemented by TextBasedChannel
-  /* eslint-disable no-empty-function */
+
+  // eslint-disable-next-line getter-return
   get lastMessage() {}
+
   send() {}
+
   sendTyping() {}
+
   createMessageCollector() {}
+
   awaitMessages() {}
+
   createMessageComponentCollector() {}
+
   awaitMessageComponent() {}
+
   bulkDelete() {}
+
   fetchWebhooks() {}
+
   createWebhook() {}
+
   setRateLimitPerUser() {}
+
   setNSFW() {}
```

Comment:
I've seen other methods like this modified to have async. Should these have it too?